x-pack/test/functional_endpoint/apps/endpoint/policy_list.ts

```diff
@@ -45,46 +45,5 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
       );
       expect(noItemsFoundMessage).to.equal('No items found');
     });
-
-    describe('and policies exists', () => {
```

Comment:
I would suggest adding a `.skip` instead of deleting the block of code.

Reply:
Why?

Reply:
Because the tests are valid, just not working properly.  Why delete them?

Reply:
agreed that it should be skipped
Why are these failing for others? Would like to understand that better

Reply:
If the code isn't running, it is dead code. Skipping code via the test framework is appropriate for debugging, but there is no reason to ship dead code to customers.

Reply:
> agreed that it should be skipped
> Why are these failing for others? Would like to understand that better

The purpose of this PR isn't to figure out the issue. It is to unblock other Kibana developers.

Reply:
To unblock due to failing/flaky tests the typical practice is to add the .skip.  This unblocks everyone while allowing the team to retain the code and fix the flakyness @oatkiller.